.github/workflows/create-update-issues.yaml

```diff
@@ -0,0 +1,76 @@
+name: Create Update Issues
+
+on:
+  workflow_call:
+    inputs:
+      AUTH_APP_ID:
+        description: Github app id to authenticate in mobile and extention repositories
+        type: string
+        required: true
+    secrets:
+      AUTH_APP_PRIVATE_KEY:
+        description: Github app private key to authenticate in mobile and extention repositories
+        required: true
+
+jobs:
+  create-update-issues:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout head
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
I'm not 100% sure that we have to fetch here whole history, if release pipeline have always tag on head commit `fetch-tags: true` to make it little faster

Reply:
I think it would be safe just to get the HEAD commit. The release tags will always be on that commit. The tags are created in the `publish-release` step, which happens before this one.

Reply:
Could publish release output created tags as it have it? This would simplify whole process. We can use also use `git tag --points-at head` command and process output it the loop